doc/release-notes-empty-template.md

```diff
@@ -35,8 +35,8 @@ wallet versions of Bitcoin Core are generally supported.
 Compatibility
 ==============
 
-Bitcoin Core is supported and tested on operating systems using the
-Linux Kernel 3.17+, macOS 13+, and Windows 10+. Bitcoin
+Bitcoin Core is supported and tested on the following operating systems or newer:
+Linux Kernel 3.17, macOS 13, and Windows 10 (version 1903). Bitcoin
```

Comment:
Is anything enforcing this, or is the assumption that any Windows 10 system is this at least version? What happens if it isn't?

Reply:
> Is anything enforcing this, or is the assumption that any Windows 10 system is at least this version?

The latter.

> What happens if it isn't?

My guess is that the application won't properly handle any symbols outside its default code page.

Reply:
Can we maybe check the codepage early at startup (e.g. `SetupEnvironment()`) and fail if it isn't correct? This seems more thorough than any version check.

Reply:
> Can we maybe check the codepage early at startup (e.g. `SetupEnvironment()`) and fail if it isn't correct? This seems more thorough than any version check.

Thanks! Done.